packages/valory/skills/decision_maker_abci/behaviours/base.py

```diff
@@ -293,17 +294,18 @@ def get_bet_amount(
     ) -> Generator[None, None, int]:
         """Get the bet amount given a specified trading strategy."""
 
-        if strategy == "bet_amount_per_conf_threshold":
+        # Kelly Criterion does not trade for equally weighted pools.
+        if (
```

Comment:
Rather than using some hack, let's just revert to this strategy when Kelly Criterion would suggest not to trade.